lms/djangoapps/teams/static/teams/templates/create-team-button.underscore

```diff
@@ -0,0 +1,4 @@
+<div class="add-team">
+  <h3 class="title">Having trouble finding a team to join?</h3>
```

Comment:
Sorry, I know this was already changed from another PR but on looking at this again, I think this version will be better for translation:
"Are you having trouble finding a team to join?"